microsoft-365/admin/add-users/let-users-reset-passwords.md

```diff
@@ -65,13 +65,17 @@ These steps turn on self-service password reset for everyone in your business.
 
 ::: moniker-end
 
-2. At the top of the Settings page select **Security & Privacy**.
+2. At the top of the **Org settings** page, select **Security & Privacy** Tab.
```

Comment:
Replace with:
2. At the top of the **Org settings** page, select the **Security & privacy** tab.